core/src/main/scala/kafka/coordinator/transaction/TransactionCoordinator.scala

```diff
@@ -278,12 +278,13 @@ class TransactionCoordinator(txnConfig: TransactionConfig,
 
   def handleListTransactions(
     filteredProducerIds: Set[Long],
-    filteredStates: Set[String]
+    filteredStates: Set[String],
+    durationFilter: Long = -1
```

Comment:
nit: this is -1 -- is that intended?

Reply:
Updated default value to -1L in other places for consistency in [81173c8](https://github.com/apache/kafka/pull/15384/commits/81173c89dbf50a98671694f99a051dad1e0b1821)

Reply:
@yyu1993 this default value needs to be changed to -1L as well

Reply:
Im not sure what he is supposed to change 😅